### DISCUSSION TITLE
Wish: performance curve of the day

### DISCUSSION THREAD

**Discussion by 2000ede** [tnmendes/watt#29] [Ideas]
Hello,

thank you for this great app!

I am particularly interested in the power function of the socket.

I can between
- week
- last 30th
- Month
- Year
choose. 

I also wish
- day.
So a performance curve of the day.

I would also like a CSV export. From the app by email.

Sorry for my bad english.

Best wishes
Ede

**Comment by tnmendes** [comment]
Hey Ede thank you for the suggestion

**Reply by 2000ede** [reply]
what is the chance that it will be implemented?
In particular, those who use the sockets for mini photovoltaic systems would be very happy. It would also be a unique selling point among power outlets.
(translated with google translate)

**Reply by tnmendes** [reply]
The CSV export is on the roadmap.

Regarding the Power, I can see the last 30 days' power consumption (that you already have charts) and the live power.

**Reply by 2000ede** [reply]
yes, that is also very good.

However, it referred to the daily curve

I already think everything else is great!

**Reply by tnmendes** [reply]
Yeah, I would also love to have a daily chart but the devices Kasa is only recording the daily sum :-/

**Reply by 2000ede** [reply]
ah, ok .... then my wish has unfortunately already burst
